modules/amxBidAdapter.js

```diff
@@ -111,6 +93,24 @@ function createBidMap(bids) {
   return out;
 }
 
+function ensureNumber(value, defaultValue) {
```

Comment:
would this be generaly useful? perhaps import it from somewhere others can access